bundles/core/org.eclipse.smarthome.core/META-INF/MANIFEST.MF

```diff
@@ -56,6 +57,11 @@ Import-Package: com.google.common.base,
  org.osgi.service.event,
  org.osgi.service.packageadmin,
  org.osgi.util.tracker,
- org.slf4j
+ org.slf4j,
+ tec.uom.lib.common,

```

Comment:
@kaikreuzer : my plan was in fact to make a POC of QuantityType in Yahoo Weather once this PR approved - but if you think it's better, I could do it in the same PR.